driver/docker/docker.go

```diff
@@ -55,11 +55,20 @@ func (d *Driver) Config() map[string]string {
 		"PULL_ALWAYS":         "Always pull image, even if locally available (0|1)",
 		"DOCKER_DRIVER_QUIET": "Make the Docker driver quiet (only print container stdout/stderr)",
 		"OUTPUTS_MOUNT_PATH":  "Absolute path to where Docker driver can create temporary directories to bundle outputs. Defaults to temp dir.",
+		"CLEANUP_CONTAINERS":  "If true, the docker container will be destroyed when it finishes running. If false, it will not be destroyed. The supported values are true and false. Defaults to true.",
 	}
 }
 
 // SetConfig sets Docker driver configuration
 func (d *Driver) SetConfig(settings map[string]string) {
+	// Set default and provide feedback on acceptable input values.
+	value, ok := settings["CLEANUP_CONTAINERS"]
+	if !ok {
+		settings["CLEANUP_CONTAINERS"] = "true"
+	} else if value != "true" && value != "false" {
+		fmt.Printf("CLEANUP_CONTAINERS environment variable has unexpected value %q. Supported values are 'true', 'false', or unset.", value)
```

Comment:
I think the cnab-go runtime shouldn't print something on the stdout without a way to disable it. `SetConfig` should either return an error (would maybe break things), or ignore invalid values and consider them as the default (aka: "true"). WDYT?

Reply:
I'd prefer to be able to return an error message, but I didn't want to change the interface for all the drivers just to add one option to one driver.

I strongly dislike UIs that involve the user setting an environment variable and then getting zero feedback if their setting is being ignored. If you have to read the code to understand why a feature doesn't seem to be working for you, then that is not a good user experience.

There are a few ways to fix this that I'd be happy with, but they all require changes to the driver interface/contract:
1) `SetConfig` could return an `error`, which would allow UX feedback for invalid config.
2) The drivers could take a stream to write their logs/errors to, and Duffle can decide whether to pass in stdout/stderr or /dev/null based on user config.
3) The drivers could all use the same logging library as Duffle, which should allow Duffle to configure global settings like log-level.

@silvin-lubecki WDYT?

Reply:
@astrieanna I'd be in favor of modifying `SetConfig` so it can return an error 👍 
But I also think we need a better way to log everything happening in the driver. That's a pain point we have in `docker app`.